stable/tomcat/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 name: tomcat
 description: Deploy a basic tomcat application server with sidecar as web archive container
 icon: http://tomcat.apache.org/res/images/tomcat.png
-version: 0.3.0
-appVersion: "7.0"
+version: 0.4.0
+appVersion: "9.0"
```

Comment:
This doesn't appear to be related and isn't mentioned in the PR description. Is the current appVersion incorrect?

Reply:
reverted